pkg/client/model_serverless_cluster_update_specification.go

```diff
@@ -21,8 +21,8 @@ package client
 // ServerlessClusterUpdateSpecification struct for ServerlessClusterUpdateSpecification.
 type ServerlessClusterUpdateSpecification struct {
 	// spend_limit is the maximum monthly charge for a cluster, in US cents. We recommend using usage_limits instead, since spend_limit will be deprecated in the future.
-	SpendLimit  *int32       `json:"spend_limit,omitempty"`
-	UsageLimits *UsageLimits `json:"usage_limits,omitempty"`
+	SpendLimit  *Nullable[int32] `json:"spend_limit,omitempty"`
+	UsageLimits *UsageLimits     `json:"usage_limits,omitempty"`
```

Comment:
Does this need to be `*Nullable[UsageLimits]`? If we didn't have the spend limit (or later remove it), then there's no way to know whether `UsageLimits=nil` means "don't set this field" vs. "unlimited".